package.json

```diff
@@ -44,6 +44,7 @@
     }
   ],
   "devDependencies": {
+    "@ontologies/core": "^2.0.3",
```

Comment:
Why dev deps for this one but not the others?

Reply:
oo I think this is a good catch, thx!